src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_ListViewBase.cs

```diff
@@ -62,6 +63,38 @@ public void InvalidSelectionChangeValidPrevious()
 			Assert.AreEqual(list.SelectedItem, 3);
 		}
 
+		// Requires access to MaterializedContainers
+		[TestMethod]
+		[RunsOnUIThread]
+		public async Task ContainerIndicesAreUpdated()
+		{
+			var source = new ObservableCollection<string>();
+			var SUT = new ListView { ItemsSource = source };
+			WindowHelper.WindowContent = SUT;
+
+			source.Add("test");
+
+			await WindowHelper.WaitForIdle();
+
+			source.Insert(0, "different");
+
+			await WindowHelper.WaitForIdle();
+
+#if HAS_UNO			
+			var containerIndices = SUT.MaterializedContainers
+				.Select(container => container.GetValue(ItemsControl.IndexForItemContainerProperty))
+				.OfType<int>()
+				.OrderBy(index => index)
+				.ToArray();			
+			CollectionAssert.AreEqual(new int[] { 0, 1 }, containerIndices);
+#else
```

Comment:
Sorry to nitpick your test so much, but it shouldn't need to be `#else` right? The remaining assertion should work on Uno (I hope)?

Reply:
Shouldn't 🤔 . I will try!

Reply:
Done :-)